### ISSUE TITLE
Filter not working in a cypher field

### ISSUE THREAD

**Issue by fatjoni** [neo4j-graphql/neo4j-graphql#116]
Hi,

I have create a type this way:
type Car  {
  registrationDate: String
  yearRegistration: Int @cypher(statement:"MATCH (this) WITH datetime(this.registrationDate) as year RETURN year.year")
  price: Float
  mileage: Float
  power: Int
  nrOfSeats: Int
  nrOfDoors: Int
  gear: Int
  bodyType: BodyType @relation(name:"BODY_TYPE",direction:OUT)
  model: Model @relation(name:"MODEL",direction:OUT)
}

It works fine when querying but when i try to filter by yearRegistration it doesnt work, it shows 0 records.
This happens when i use the filter method or when i filter directly by attribute.

Thanks in advance

**Comment by jexp** [comment]
yes dynamic fields are different, we have to figure out how to integrate them in a better way.
